File: LNPG_01_TIPOS_E_VARIAVEIS/Java/src/maior_ou_menor_de_idade_3.java
import java.util.Scanner;

public class maior_ou_menor_de_idade_3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Insira a sua idade.\n");
        int age_user = scanner.nextInt();

        if (age_user >= 18) {
            System.out.printf("Voce e maior de idade, pois sua idade (%d) e superior a 17.", age_user);
        } 
        else {
            System.out.printf("Voce e menor de idade, pois sua idade (%d) e inferior a 18.", age_user);
        }

        scanner.close();
    }
}

File: LNPG_01_TIPOS_E_VARIAVEIS/C/maior_ou_menor_de_idade_3.c
#include <stdio.h>

int main() {
    int age_user;
    printf("Insira a sua idade (ex.: 20)\n");
    scanf("%d", &age_user);
    
   if (age_user >= 18) {
       printf("Voce e maior de idade, pois sua idade (%d) e superior a 17.", age_user);
   }
   else {
       printf("Voce e menor de idade, pois sua idade (%d) e inferior a 18.", age_user);
   }
   
    return 0;
}